frontend/src/pages/TeacherProfile.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <Page blueBg>
+    <div class="teacher-profile-container">
+      <div class="teacher-profile-left">
+        <StudentProfileCard
+          v-bind:studentInfo="{
+            studentName: teacherData.FullName,
+            studentContact: teacherData.PhoneNumber,
+            source: teacherData.Source,
+            dateJoined: teacherData.dateJoined,
+            nativeLanguage: teacherData.nativeLanguage.NativeLanguage,
+            notes: teacherData.Notes,
+            status: teacherData.status.Description,
+            proficiencyLevel: teacherData.EnglishProficiency,
+          }"
+          v-bind:teacherInfo="{
+            secondLanguage: teacherData.secondLanguage.NativeLanguage,
+            languageProficiency: teacherData.LanguageProficiency,
+          }"
+          v-bind:isTeacher="true"
+          v-on:englishProficiencyIsSelected="englishProficiency = $event"
+        />
+      </div>
+      <div class="teacher-profile-right">
+        <section class="student-status-section">
+          <StatusCardScreening
+            :studentID="teacherID"
+            :englishProficiency="englishProficiency"
+            v-if="teacherData.status.Description.toUpperCase() === 'SCREENING'"
+          />
+          <StatusCardUnmatched
+            :studentID="teacherID"
+            v-if="teacherData.status.Description.toUpperCase() === 'UNMATCHED'"
+          />
```

Comment:
Unlike in the `StudentProfile.vue`, in `TeacherProfile.vue` we should only be able to select "Drop out" for an unmatched Teacher. We shouldn't be able to select "Select Match". See screenshots below from Figma. So we'll need to switch on `isTeacher` in the `StatusCardUnmatched` component. 

![image](https://user-images.githubusercontent.com/19401055/105568606-b5f19f80-5d75-11eb-878d-f98f9c8584f6.png)

![image](https://user-images.githubusercontent.com/19401055/105568616-c275f800-5d75-11eb-8956-1e96694b177a.png)

Reply:
Thanks, we made the changes to the cards!